Makefile

```diff
@@ -7,7 +7,10 @@ fmt: ## Run go fmt against code
 
 .PHONY: test
 test: ## Run go test against code
-	go test ./pkg/... ./cmd/...
+	docker-compose build $@ && docker-compose run --rm $@ make _$@
```

Comment:
I just found that this repository uses Prow (no CI config files in the repo, had to do the PR to know that), with a src image of Openshift.
You are right it can't run docker-compose at the moment.

Eventually it depends on the image being used in the PR (which can get changed). 
But for now - I will keep the Makefile targets the same. Although I won't be able to add my tests to be a part of the CI. It depends on having `cap_net_admin` capabilities.